1001-2000/LIT1510GadlaY.xml

```diff
@@ -50,11 +61,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
          <change who="DR" when="2017-03-09">Added keywords and updated title</change>
+         <change who="MKr" when="2023-08-16">Added bibliography, clavis, relation, abstract</change>
       </revisionDesc>
    </teiHeader>
+   <text>
+    <body>
+        <div type="bibliography">
+          <listBibl type="editions">
+              <bibl><ptr target="bm:Heide2000Testamente"/></bibl>
+          </listBibl>
+            <listRelation>
+                <relation name="ecrm:P129_is_about" active="LIT1510GadlaY" passive="PRS5650Jacob"/>
+            </listRelation>
```

Comment:
do we need an ID for the collection of testaments?
LIT1412GadlaA
LIT1510GadlaY
LIT1517GadlaY

and then a relation?